### PR TITLE
ENT-8496: Added page for highlighting some editor support

### PR DESCRIPTION
**Pull request by nickanderson**
Ticket: ENT-8496
Changelog: None

### CODE REVIEW

**Comment by nickanderson** [comment]
@cf-bottom jenkins, please

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/7767/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/7767/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-7767/

**Documentation:** http://buildcache.cfengine.com/packages/build-documentation-pr/jenkins-pr-pipeline-7767/output/_site/

**Comment by nickanderson** [comment]
@cf-bottom jenkins, please

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/7770/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/7770/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-7770/

**Documentation:** http://buildcache.cfengine.com/packages/build-documentation-pr/jenkins-pr-pipeline-7770/output/_site/

File: guide/writing-and-serving-policy/editors.markdown [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+layout: default
+title: Editors
+published: true
+sorting: 10
+tags: [tools, editor, vim, emacs, vscode, kate, sublime text, atom, eclipse ]
+---
+
+Using an editor that provides syntax highlighting and other features can significantly enhance prodcutivity and quality of life.
+
+## Emacs ##
```
Comment (olehermanse) [inline guide/writing-and-serving-policy/editors.markdown:11]:
```suggestion
## Emacs
```

**Review by olehermanse — APPROVED** [review]
LGTM

**Pull request merged**